### PR TITLE
chore: Adjust add and delete icon colors to match server variables

### PR DESCRIPTION
**Pull request by stefan-niedermann**
:information_source: These colors are still hard coded. I could imagine adding a `<style>` tag to the `svg` icons to actually apply the colors based on the available variables. However this is some kind of paradigm change in handling `svg` icons, therefore I wanted to keep the change small.

Source of color codes: https://github.com/nextcloud/server/blob/1891ae121d80758ffaf537dd4d4fffbaed3eef00/apps/theming/css/default.css#L24-L32

CC @nextcloud/designers

### CODE REVIEW

File: img/add-color.svg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <svg xmlns="http://www.w3.org/2000/svg" height="16" width="16" version="1.1" viewBox="0 0 16 16">
- <g transform="matrix(-.70711 -.70711 .70711 -.70711 -724.85 753.16)" fill="#3ea857">
-  <path d="m3.7547 1041.6 1.4142-1.4142 3.5355 3.5355 3.5355-3.5355 1.4142 1.4142-3.5355 3.5355 3.5355 3.5356-1.4142 1.4142-3.5355-3.5356-3.5164 3.5547-1.4333-1.4333 3.5355-3.5356z" fill="#3ea857"/>
+ <g transform="matrix(-.70711 -.70711 .70711 -.70711 -724.85 753.16)" fill="#2d7b41">
```
[the remark this answers is not in the document]
Comment (stefan-niedermann) [reply img/add-color.svg:3]: Sure, see the description of this PR, I suggested it (and would be willing to contribute) if you are fine with using `style` tags and variables within the `svg` files 😉

[the remark this answers is not in the document]
Comment (stefan-niedermann) [reply img/add-color.svg:8]: Might be useful as fallbacks for missing CSS variable, browsers not being able to render SVG with style tags correctly etc.

All of those scenarios are unlikely, it just looked like a cheap safety net 🤷

**Review by emoral435 — APPROVED** [review]

File: img/add-color.svg [hunk 2]
```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <svg xmlns="http://www.w3.org/2000/svg" height="16" width="16" version="1.1" viewBox="0 0 16 16">
- <g transform="matrix(-.70711 -.70711 .70711 -.70711 -724.85 753.16)" fill="#3ea857">
-  <path d="m3.7547 1041.6 1.4142-1.4142 3.5355 3.5355 3.5355-3.5355 1.4142 1.4142-3.5355 3.5355 3.5355 3.5356-1.4142 1.4142-3.5355-3.5356-3.5164 3.5547-1.4333-1.4333 3.5355-3.5356z" fill="#3ea857"/>
+ <style>
+   g, path {
+     fill: var(--color-success, #2d7b41);
+   }
+ </style>
+ <g transform="matrix(-.70711 -.70711 .70711 -.70711 -724.85 753.16)" fill="#2d7b41">
```
Comment (ShGKme) [inline img/add-color.svg:8]:
```suggestion
 <g transform="matrix(-.70711 -.70711 .70711 -.70711 -724.85 753.16)" fill="var(--color-success, #2d7b41)">
```

File: img/add-color.svg [hunk 3]
```diff
@@ -1,6 +1,11 @@
...
+  <path d="m3.7547 1041.6 1.4142-1.4142 3.5355 3.5355 3.5355-3.5355 1.4142 1.4142-3.5355 3.5355 3.5355 3.5356-1.4142 1.4142-3.5355-3.5356-3.5164 3.5547-1.4333-1.4333 3.5355-3.5356z" fill="#2d7b41"/>
```
Comment (ShGKme) [inline img/add-color.svg:9]:
```suggestion
  <path d="m3.7547 1041.6 1.4142-1.4142 3.5355 3.5355 3.5355-3.5355 1.4142 1.4142-3.5355 3.5355 3.5355 3.5356-1.4142 1.4142-3.5355-3.5356-3.5164 3.5547-1.4333-1.4333 3.5355-3.5356z" fill="var(--color-success, #2d7b41)"/>
```

File: img/delete-color.svg [hunk 4]
```diff
@@ -1,4 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <svg xmlns="http://www.w3.org/2000/svg" height="16" width="16" version="1.1" viewBox="0 0 16 16">
- <path d="m12.95 11.536-1.414 1.414-3.536-3.5358-3.5355 3.5358-1.4142-1.414 3.5355-3.536-3.5355-3.5356 1.4142-1.4142 3.5355 3.5356 3.516-3.5547 1.434 1.4333-3.5357 3.5356z" fill="#d40000"/>
+ <style>
+   g, path {
+     fill: var(--color-error, #DB0606);
+   }
+ </style>
+ <path d="m12.95 11.536-1.414 1.414-3.536-3.5358-3.5355 3.5358-1.4142-1.414 3.5355-3.536-3.5355-3.5356 1.4142-1.4142 3.5355 3.5356 3.516-3.5547 1.434 1.4333-3.5357 3.5356z" fill="#DB0606"/>
```
Comment (ShGKme) [inline img/delete-color.svg:8]:
```suggestion
 <path d="m12.95 11.536-1.414 1.414-3.536-3.5358-3.5355 3.5358-1.4142-1.414 3.5355-3.536-3.5355-3.5356 1.4142-1.4142 3.5355 3.5356 3.516-3.5547 1.434 1.4333-3.5357 3.5356z" fill="var(--color-error, #DB0606)"/>
```

**Review by ShGKme — APPROVED** [review]
To simplify SVG a bit. We have the same attributes changes in other places.

**Comment by stefan-niedermann** [comment]
Uh, I didn't know one can use CSS custom properties directly in `fill` attributes. I would have expected at least a `style` attribute. :)

**Comment by ShGKme** [comment]
> Uh, I didn't know one can use CSS custom properties directly in `fill` attributes. I would have expected at least a `style` attribute. :)

Me too to be honest, but we have done it in many places recently 😀

**Review by ShGKme — APPROVED** [review]

**Pull request merged**